docs/docs/manual/installing.md

```diff
@@ -93,16 +93,16 @@ import TabItem from '@theme/TabItem';
 
 <TabItem value="win">
 
-1. On Windows 10, click the Windows start menu button, type "env," and look at the search results. Click “Edit the system environment variables.” (If you are using an earlier version of Windows, use the “Search” or “Search programs and files” box in the start menu.)
+1. On Windows 10, click the Windows start menu button, type <span class="menuItems">env</span>, and look at the search results. Click <span class="menuItems">Edit the system environment variables</span>. (If you are using an earlier version of Windows, use the <span class="menuItems">Search</span> or <span class="menuItems">Search programs and files</span> box in the start menu.)
```

Comment:
Many things in OpenRefine can be clicked, such as Buttons, Menu Items, etc.
I think it's fine for now to have this separate style for **menuItems** directly and where we can always add additional style groups for other element types like Buttons (OK, Cancel, X-close, - minimize, Open, etc.) and have iconography added for them as well, if necessary in the future.  According to Docusaurus docs, this can be done with images and svgs https://v2.docusaurus.io/docs/markdown-features/#images